lib/Text/Glob.pm

```diff
@@ -177,6 +189,10 @@ the regex, or change the seperator that Text::Glob uses by setting
 C<$Text::Glob::seperator> to an alternative value while compiling the
 the regex.
 
+However, if C<$Text::Glob::globstar> is set to a true value, two
+consecutive asterisks (e.g. C<**.foo>) B<will> match C</>, though one
+asterisk will not.
+
```

Comment:
>Looking at it now I can imagine a further tweak to collapse consecutive slashes

@trwyant I've added you as a collaborator to the "mithun/perl-text-glob" fork. Can you please assist with the change? I can then help document/cleanup/squash.